src/unistd.rs

```diff
@@ -1,47 +1,53 @@
 //! Safe wrappers around functions found in libc "unistd.h" header
 
-#[cfg(not(target_os = "redox"))]
-use cfg_if::cfg_if;
```

Comment:
It looks like there are some style changes in this section.  Could you please submit style changes in separate commits from content changes (but not necessarily separate PRs).

Reply:
Sure. My SOP is to have many small commits. But I was asked to squash everything from my prior PR, so I figured that was preferred. Was there something in the CONTRIBUTING doc that would have told me beforehand?

Reply:
This is now a second commit, and is severable if you prefer. It contains no functional changes.